posix.mak

```diff
@@ -251,6 +251,15 @@ MAKEFILE = $(firstword $(MAKEFILE_LIST))
 # build with shared library support (defaults to true on supported platforms)
 SHARED=$(if $(findstring $(OS),linux freebsd),1,)
 
+# Check independent executability of public unittests on dlang.org
+IGNORED_PUBLICTESTS= $(addprefix std/, \
```

Comment:
I think this needs a more detailed comment of what this does and why it's needed